src/main/java/com/gtnewhorizon/structurelib/proxy/ClientProxy.java

```diff
@@ -138,6 +138,18 @@ public void endHinting(World w) {
         currentHints = null;
     }
 
+    @Override
+    public void clearHints(World w) {
+        if (!w.isRemote) {
+            if(currentHints!=null) {
+                currentHints.forEach(EntityFXBlockHint::setDead);
```

Comment:
This doesn't look right. Merge conflict?

Reply:
This works so...

Reply:
actually, I just noticed there is indeed some merge conflicts to resolve. It would work alright on your branch, but would have compile errors if it is merged into master.